src/wazuh_mcp_server/api/wazuh_indexer_client.py

```
@@ -128,7 +128,7 @@ def _validate_ssl_config(self):
                             "recommendations": detection_results.get("recommendations")
                         }
                     })
-            except Exception as e:
+            except Exception:
                 logger.debug(f"SSL auto-detection failed: {e}")
```

Ruff (0.11.9)

Undefined name `e`

(F821)